src/Controls/tests/DeviceTests/ControlsHandlerTestBase.cs

```diff
@@ -195,6 +195,9 @@ protected Task CreateHandlerAndAddToWindow<THandler>(IElement view, Func<THandle
 
 						if (!isDestroyed)
 							window.Destroying();
+
+						if (window is Window win)
+							win.Page = null;
```

Comment:
I think maybe one of the changes here cause the Android tests to crash (maybe this change ☝️ ):
```
      INSTRUMENTATION_RESULT: longMsg=android.runtime.JavaProxyThrowable: System.InvalidOperationException: No page was set on the window.
         at Microsoft.Maui.Controls.Window.Microsoft.Maui.IWindow.get_Content()
         at Microsoft.Maui.Handlers.WindowHandler.CreateRootViewFromContent(IWindowHandler handler, IWindow window)
         at Microsoft.Maui.DeviceTests.Stubs.WindowHandlerStub.UpdateContent()
         at Microsoft.Maui.DeviceTests.Stubs.WindowHandlerStub.MapContent(WindowHandlerStub handler, IWindow window)
         at Microsoft.Maui.PropertyMapper`2.<>c__DisplayClass5_0[[Microsoft.Maui.IWindow, Microsoft.Maui, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null],[Microsoft.Maui.DeviceTests.Stubs.WindowHandlerStub, Microsoft.Maui.Controls.DeviceTests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]].<Add>b__0(IElementHandler h, IElement v)
         at Microsoft.Maui.PropertyMapper.UpdatePropertyCore(String key, IElementHandler viewHandler, IElement virtualView)
         at Microsoft.Maui.PropertyMapper.UpdateProperties(IElementHandler viewHandler, IElement virtualView)
         at Microsoft.Maui.Handlers.ElementHandler.SetVirtualView(IElement view)
         at Microsoft.Maui.Controls.Element.SetHandler(IElementHandler newHandler)
         at Microsoft.Maui.Controls.Element.set_Handler(IElementHandler value)
         at Microsoft.Maui.Platform.ElementExtensions.ToHandler(IElement view, IMauiContext context)
         at Microsoft.Maui.DeviceTests.ControlsHandlerTestBase.WindowTestFragment.OnCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState)
         at AndroidX.Fragment.App.Fragment.n_OnCreateView_Landroid_view_LayoutInflater_Landroid_view_ViewGroup_Landroid_os_Bundle_(IntPtr jnienv, IntPtr native__this, IntPtr native_inflater, IntPtr native_container, IntPtr native_savedInstanceState)
         at Android.Runtime.JNINativeWrapper.Wrap_JniMarshal_PPLLL_L(_JniMarshal_PPLLL_L callback, IntPtr jnienv, IntPtr klazz, IntPtr p0, IntPtr p1, IntPtr p2)
      INSTRUMENTATION_CODE: 0
```

Looking into it locally.